	do not try to set user or group id when installing
--- proftpd-1.3.3.orig/Make.rules.in	2009-11-18 22:48:25.000000000 +0100
+++ proftpd-1.3.3/Make.rules.in	2010-05-30 12:47:38.265625000 +0200
@@ -28,9 +28,9 @@ UTILS_LIBS=@UTILS_LIBS@ @LIBS@
 INSTALL=@INSTALL@
 INSTALL_USER=@install_user@
 INSTALL_GROUP=@install_group@
-INSTALL_BIN=$(INSTALL) @INSTALL_STRIP@ -o $(INSTALL_USER) -g $(INSTALL_GROUP) -m 0755
-INSTALL_SBIN=$(INSTALL) @INSTALL_STRIP@ -o $(INSTALL_USER) -g $(INSTALL_GROUP) -m 0755
-INSTALL_MAN=$(INSTALL) -o $(INSTALL_USER) -g $(INSTALL_GROUP) -m 0644
+INSTALL_BIN=$(INSTALL) -m 0755
+INSTALL_SBIN=$(INSTALL) -m 0755
+INSTALL_MAN=$(INSTALL) -m 0644
 
 RM=rm -f
 SHELL=@CONFIG_SHELL@
